Dockerfile

```diff
@@ -0,0 +1,41 @@
+# To build:
```

Comment:
This would probably be good to have in the top level README as well.

Reply:
I agree, though I think it is good enough to merge for now.